hw/top_earlgrey/formal/conn_csvs/ast_clkmgr.csv

```diff
@@ -0,0 +1,37 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Run these checks with:
+#  ./util/dvsim/dvsim.py hw/top_earlgrey/formal/chip_conn_cfg.hjson
+
+,NAME,SRC BLOCK,SRC SIGNAL,DEST BLOCK,DEST SIGNAL,,,,,,
+
+#################################
+# Signals from AST to Clkmgr
+#################################
+CONNECTION, AST_CLK_SYS_OUT,     u_ast, clk_src_sys_o,     top_earlgrey.u_clkmgr_aon,     clk_main_i
+CONNECTION, AST_CLK_AON_OUT,     u_ast, clk_src_aon_o,     top_earlgrey.u_clkmgr_aon,     clk_aon_i
+CONNECTION, AST_CLK_USB_OUT,     u_ast, clk_src_usb_o,     top_earlgrey.u_clkmgr_aon,     clk_usb_i
+CONNECTION, AST_CLK_IO_OUT,      u_ast, clk_src_io_o,      top_earlgrey.u_clkmgr_aon,     clk_io_i
+
+CONNECTION, AST_ALL_BYP_ACK_OUT, u_ast, all_clk_byp_ack_o, top_earlgrey.u_clkmgr_aon, all_clk_byp_ack_i
+CONNECTION, AST_IO_BYP_ACK_OUT,  u_ast, io_clk_byp_ack_o,  top_earlgrey.u_clkmgr_aon, io_clk_byp_ack_i
+
+#################################
+# Signals from Clkmgr to AST
+#################################
+# TODO: why is this clk_aon_secure in the chip-level, when the module itself is connected to clk_aon_peri?
```

Comment:
we should change the chip-level assignment from `clk_aon_secure` -> `clk_aon_peri`

Reply:
done.